src/Workleap.DomainEventPropagation.Abstractions/Workleap.DomainEventPropagation.Abstractions.csproj

```diff
@@ -36,6 +36,8 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
```

Comment:
I think ideally we want to keep the abstractions project as light as possible. Now it's a bit weird there would be registration logic when this package is meant to be referenced in projects where you usually won't be doing registrations. We could probably just have it as a shared file like the others 🤔or are we at the point where we should have a project like `Workleap.DomainEventPropagation.Common` @asimmon

Reply:
J'ai jasé avec Antho, mettre ça dans le folder Shared est good. Donc retirer les ServiceCollectionExtensions du projet Abstractions et mettre dans Shared 🙂